### PR TITLE
Add static diagram

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Add] Static diagram and build config
    - Only most recent https://www.npmjs.com/package/@svgdotjs/svg.js used, not deprecated https://www.npmjs.com/package/svgjs
    - All static diagram packages are marked dev dependencies
    - SVG files are currently not shared between static and interactive, for future improvement
    - Static and interactive diagrams share `output` directory. `dist` is only used for interactive tester

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,9 +27,12 @@
...
     "scripts": {
-        "start": "parcel clearcalcs/**/index.html",
-        "test": "parcel clearcalcs/**/*.html --port 4321",
-        "compile": "parcel build clearcalcs/**/index.html",
-        "docs": "cd docs && npm install && docsify serve -p 4444"
-    }
+        "start": "parcel clearcalcs/interactive/**/index.html",
+        "test": "parcel clearcalcs/interactive/**/*.html --port 4321",
+        "compile-interactive": "parcel build clearcalcs/interactive/**/index.html",
+        "docs": "cd docs && npm install && docsify serve -p 4444",
```
Comment (JulianGoh5) [inline package.json:33]: Nice to see this in here! Would be useful for spinning up the docs readily

File: src/static/interface.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import update from "./render";
+
+render = function (params, storedParams) {
+    return update(params, storedParams);
+};
+
+params = function () {
```
Comment (JulianGoh5) [inline src/static/interface.ts:7]: Surprises me that these are fine without a `let` or `const` in front of them

Comment (JulianGoh5) [reply src/static/interface.ts:7]: Ah it turns out that implicitly these become global variables

**Review by JulianGoh5 — APPROVED** [review]

File: src/static/interface.ts (hunk 2 above)
Comment (michaellisitsa) [reply src/static/interface.ts:7]: Typescript still complains here though

**Pull request merged**